Bottom: adb49a0b0a2ec3b4c9062a315af406282538f18c
Top:    804b796934b13001b2481ca5c87e0ea19081d001
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-21 15:09:29 +0000

More debug stuff


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 8de425d..00c6f90 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2960,6 +2960,8 @@ M_Set_Parentage (POLYAREA *poly, POLYPARENTAGE parentage)
 
 }
 
+static void clear_marks (POLYAREA * p);
+
 /* just like poly_Boolean but frees the input polys */
 int
 poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
@@ -2971,13 +2973,12 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
   int code;
   CVCList *the_list;
   POLYAREA *a_copy, *b_copy;
+  CVCList *the_list;
 
   *res = NULL;
 
   g_warning ("BEGIN BOOLEAN");
 
-  *res = NULL;
-
   /* Make copies for tracking polygon parentage (DEBUG) */
   if (!poly_M_Copy0 (&a_copy, a) || !poly_M_Copy0 (&b_copy, b))
       return err_no_memory;
@@ -2994,7 +2995,6 @@ poly_Boolean_free (POLYAREA * ai, POLYAREA * bi, POLYAREA ** res, int action)
       M_Set_Parentage (b_copy, b->parentage);
       M_Set_Parentage (b, no_parentage);
     }
->>>>>>> patched
 
   if (!a)
     {
